web_src/css/base.css

```diff
@@ -944,44 +945,44 @@ img.ui.avatar,
   }
 }
 
-.following.bar {
-  margin: 0 !important;
-}
-
-.following.bar.light {
-  background: var(--color-header-bar);
+#navbar {
+  display: flex;
+  align-items: center;
+  background: var(--color-nav-bg);
   border-bottom: 1px solid var(--color-secondary);
+  width: 100vw;
+  min-height: 52px;
+  margin: 0 !important;
 }
 
-.following.bar .column .menu {
+#navbar .column .menu {
   margin-top: 0;
 }
 
-.following.bar .fitted .svg {
+#navbar .fitted .svg {
   margin-right: 0;
   vertical-align: middle;
 }
 
-.following.bar .searchbox {
+#navbar .searchbox {
```

Comment:
There seems no `searchbox` in code.

Reply:
Yes, this searchbox is commented out in templates as a TODO, we can clean this up.